ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -205,19 +213,20 @@ compilerDependencies ghcFlavor =
     , "primop-vector-tys.hs-incl"
     , "primop-vector-uniques.hs-incl"
     ] ++
-    [ "primop-docs.hs-incl" | ghcFlavor == GhcMaster ]
+    [ "primop-docs.hs-incl" | isGhcMaster ghcFlavor ]
```

Comment:
While this is not longer, I still don’t find it any easier to read.

Reply:
ok. let's get the ord thing going and not mess with the `==` cases.

Reply:
We agree to not change this and leave it as `==`.